sets.c

```diff
@@ -0,0 +1,222 @@
+/*
+    Create sets.
+
+    This file is part of libscanmem.
+
+    This library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published
+    by the Free Software Foundation; either version 3 of the License, or
+    (at your option) any later version.
+
+    This library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with this library.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+#include <ctype.h>
+#include <sys/types.h>
+
+#include "value.h"
+#include "sets.h"
+
+#define DEFAULT_UINTLS_SZ (1000)
+
+/*
+ * XXX: this should be in a utility file of some sort (no pun intended)
```

Comment:
So you wanted this function in `sets.h`?

Reply:
No, I'm saying that this comment made sense when the whole thing was in `value.{h,c}`.
Now that the set functions have their own file, this commit is unnecessary.

Reply:
I was originally thinking it could be made a generic sort function for `qsort()` if we wanted to use that anywhere else in `libscanmem`. If you doubt we will use `qsort()` elsewhere, indeed I could leave it here.

Reply:
Mah, leave it where it is, maybe just change the comment to say this function is used in `qsort()` only (though the signature alone is probably enough for anyone familiar with `qsort()`).

Reply:
Updated